src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -103,6 +106,20 @@ public void addPerson(ReadOnlyPerson person) throws DuplicatePersonException {
         }
 
         @Override
+        public void addEvent(ReadOnlyEvent event) throws DuplicateEventException {
```

Comment:
No need to include them here since they are already written in the `ModelStub` inner class of `AddEventCommandTest`